### ISSUE TITLE
Error when whitespace in project path

### ISSUE THREAD

**Issue by MrSinamun** [Microsoft/vscode-maven#304]
**Describe the bug**
When trying to open the MAVEN PROJECTS > Plugins tab within my project there is the following error message: 
```
Spawn {"command":"mvn","args":["help:effective-pom","-Doutput=\"c:\\Users\\Mr Sinamun\\AppData\\Roaming\\Code\\User\\workspaceStorage\\7acaa3e44d8a285ba3a632c9cfa8f64b\\vscjava.vscode-maven\\2878fcfd49346ba9c689d26d3cf16622\"","-f","c:\\Users\\Mr Sinamun\\IdeaProjects\\ps_18_gruppe42\\pom.xml"]}
POM file c:\Users\Mr Sinamun specified the -f/--file command-line argument does not exist 
```
I think this happens because my project resides in a path with a space:` ...\\Mr Sinamun\\....`
This bug does not happen when the project is moved to `c:\`.

**To Reproduce**
1. Open a Maven Project within a path with space. (e. g. `C:\Users\Mr Sinamun\myproject`)
2. Go to 'Maven Projects'
3. Click on 'Plugins'

**Expected behavior**
The MAVEN PROJECTS > Plugins tab should open even when the project resides in a path with a space.

**Environments**
 - OS: Windows 10
 - VS Code version: 1.33 (user setup)
 - Extension version 0.16.0
 - Maven 3.6.0

**Comment by Eskibear** [comment]
the fix has been shipped with v0.16.1, please have a try.

**Comment by MrSinamun** [comment]
It now works perfectly - Thank you for your work
